lib/Remote/Service/DriverService.php

```diff
@@ -152,6 +152,27 @@ protected function checkWasStarted($process)
         }
     }
 
+    /**
+     * @throws RuntimeException
+     */
+    protected static function checkPortIsAvail(int $port)
+    {
+        $errTest = static function ($sock) use ($port) {
```

Comment:
This method should check if the Sockets extension is enabled, and if not, it should just return true (ie. the same behavior as before - without free port detection).